app/models/post.rb

```diff
@@ -1,2 +1,9 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :comments
+  has_many :likes
```

Comment:
- [ Required ] I think it would be nice if you add a method that `updates` the posts counter for a user as suggested by the previous reviewer. Currently, the `Post` model does not have this method. Please see the example below for a suggestion on how to achieve this.

## Example code
```ruby
  def update_post_counter
    author.increment!(:posts_counter)
  end
```